meeting-notes/security-registry-wg/2020-03-25.md

```diff
@@ -1,6 +1,6 @@
 |            |                                                                   |
 | ---------- | ----------------------------------------------------------------- |
-| Recording  | TBA                                                               |
+| Recording  | https://youtu.be/wn2-oWx4qDw                                                               |
```

Comment:
Is this live yet?
I get this message when I follow that URL:
 
```
Premiere will begin shortly
29 June 2020 at 18:09
```

Reply:
still uploading *shakes fist at internet gods* but the URL is set and should work once it's up

Reply:
feel free to check back tomorrow. i'm uploading a batch of videos right now.

Reply:
The recording is up now, thanks a lot!